pkgs/build-support/buildenv/builder.pl

```diff
@@ -126,7 +126,7 @@ sub findFiles {
     return if
         $relName eq "/propagated-build-inputs" ||
         $relName eq "/nix-support" ||
-        $relName =~ /info\/dir/ ||
```

Comment:
used to include $ https://github.com/nixos/nixpkgs/commit/a6fcf45726bc75c0c7014ac6b5bedd56d649ca08#diff-1967dcc359700a88592924a2e08cc397d4c8ccedec0717a5f96bafc33ece5e53L61

but it was removed in 2010 https://github.com/nixos/nixpkgs/commit/a6fcf45726bc75c0c7014ac6b5bedd56d649ca08#diff-1967dcc359700a88592924a2e08cc397d4c8ccedec0717a5f96bafc33ece5e53R53

guessing he accidentally removed it when he changed the others to eq, while this was intentionally not changed

Reply:
Thanks. Now I'm wondering why it shouldn't be `$relName eq "/share/info/dir"` unless there may be a package with info files under some other prefix.

Reply:
I checked with nix-index and there's some that aren't under toplevel share